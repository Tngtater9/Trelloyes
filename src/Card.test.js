import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card';



describe("<Card />", () => {
    it("checks to see if Card with props crashes", () => {
        const container = document.createElement('div');
        ReactDOM.render(<Card title="some title" content="some content" />, container);

        ReactDOM.unmountComponentAtNode(container);
    })
    it("checks snapshot of Card",() => {
        const tree = renderer
        .create(<Card title="some title" content="some content" />)
        .toJSON();
        expect(tree).toMatchSnapshot();
    })
} )