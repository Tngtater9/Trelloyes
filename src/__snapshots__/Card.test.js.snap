// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> checks snapshot of Card 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    some title
  </h3>
  <p>
    some content
  </p>
</div>
`;
